src/entry/controller/item.ts

```diff
@@ -6,39 +6,28 @@ import * as entryService from '../service';
 import * as personService from '../../person/service';
 import * as projectService from '../../project/service';
 import * as hal from '../formats/hal';
-
-import { Entry as EntrySchema } from '@badgateway/tt-types';
+import {DateTime} from 'luxon';
+import {Entry as EntrySchema} from '@badgateway/tt-types';
 
 class Entry extends Controller {
-
   async get(ctx: Context) {
-
     ctx.response.type = 'application/hal+json';
 
-    const person = await personService.findById(
-      +ctx.params.personId,
-    );
+    const person = await personService.findById(+ctx.params.personId);
 
     ctx.response.body = hal.item(
-      await entryService.findById(
```

Comment:
I noticed your PRs are doing a lot of unrelated changes, which makes it a bit hard to figure out what was actually changed. I assume this is some kind of auto-formatter?

Reply:
**oh gosh** yes I didn't catch that at all this time, have been using Prettier since working on the previous client

Reply:
Makes sense. If you love prettier, i think it's better to apply it wholesale in a separate PR. If you can take it or leave it, I'd vote for no prettier ;)

Reply:
I think no prettier is better in this case 100%